staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -131,6 +131,7 @@ type JobSpec struct {
 // JobStatus represents the current state of a Job.
 type JobStatus struct {
 	// The latest available observations of an object's current state.
+	// When a job fails, one of the conditions will have type == "Failed".
```

Comment:
Please update both places, even though this is the canonical one :)

Also you'll need to regenerate the generated files. The "verify" test suite is not flaky, if it's failing something needs to be fixed :)

Reply:
Alright. I will do that now